sources/packages/web/src/components/common/notes/CreateNoteModal.vue

```diff
@@ -20,7 +20,7 @@
           :items="noteTypeItems"
           v-model="formModel.noteType"
           variant="outlined"
-          :rules="[(v) => !!v || 'Note type is required.']" />
+          :rules="[(v) => v !== null || 'Note type is required.']" />
```

Comment:
Why was it changed to check only null? Is it also validating empty strings?
Now that we have the composable, the required can be created there as a common function and accept the field name as a parameter to generate a common message.